### DISCUSSION TITLE
add a way to increase stamina pool through skills and actions

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2697] [General]
it'd be nice to be able to increase stamina pool. some possible ways to increase them:
1. train skills like swimming or melee
2. total tiles walked/ran/crouched/swam

**Comment by ghost** [comment]
Would like to see the ability to tie more variables to a skill, so that mods could add a new Fitness skill. Example below. We have support for it being a method of attack, but if we could tie stat increases including stamina in other JSON fields that would work, then we use an in-house mod similar to Stats Through Skills for this.

`"time_to_attack": { "min_time": 20, "base_time": 100, "time_reduction_per_level": 7 },
	"companion_survival_rank_factor": 1,
	"companion_skill_practice": [ { "skill": "hunting", "weight": 1 }, { "skill": "combat", "weight": 1 } ]`
